CHANGELOG.md

```diff
@@ -5,6 +5,59 @@ All notable changes to Zebra are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.3.0](https://github.com/ZcashFoundation/zebra/releases/tag/v1.3.0) - 2023-09-25
+
+This release provides support for the Spend before Sync synchronization
+strategy, and contains the following updates:
+
+### Security
+
+- Avoid panics and history tree consensus database concurrency bugs (#7590)
+
+### Added
+
+- Add missing elasticsearch flag feature to lib docs (#7568)
+- Add self hosted runner for long tests (#7520)
+- Add missing Docker variables and examples (#7552)
+
+#### Spend before Sync support
+
+- Add state requests and support code for the `z_getsubtreesbyindex` RPC (#7408)
+- Implement the `z_getsubtreesbyindex` RPC (#7436)
+- Add snapshot tests for the `z_getsubtreesbyindex` RPC (#7514)
+- Add fixed test vectors for `z_getsubtreesbyindex` from zcashd to zebra ([#7515]
+- Test `z_getsubtreesbyindex` using a lightwalletd gRPC request (#7521)
```

Comment:
These tests aren't visible to users, but we might want to mention that we tested the code. How about a single line that links to all 3 PRs?

Reply:
Done in 4e9ee83e103c67d26d02bb990722b11f9ec66f1d.